pkgs/tools/system/kmon/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, rustPlatform, python3, libxcb }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kmon";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "orhun";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1zsgyigr6k0zqhhv0icpvvfij7r7fj0l2f77i5zh1vyb89pni50h";
+  };
+
+  cargoSha256 = "15lj6ryll24xdmv8gil04prhcri2d3b1rxqjrwx70blfrlvvhxa8";
+
+  verifyCargoDeps = true;
```

@bhipple does this do anything?
```suggestion
  verifyCargoDeps = true;
```

---

It does not; it's now the default, so you can delete it entirely.  (I've already removed its reference from the manual, and there are 0 calls to it in the NixPkgs tree now.)

---

Removed.